### DISCUSSION TITLE
How to use CefSharp with Ungoogled Chromium?

### DISCUSSION THREAD

**Discussion by zxcvqwerasdf** [cefsharp/CefSharp#4824] [Q&A]
There is any patches in chromium to work with CefSharp? Because after i replaced native libs from ungoogled chromium, cefsharp isn't initializing (version is correct)

**Comment by amaitland** [comment]
> How to use CefSharp with Ungoogled Chromium?

Not supported.

You'd need to compile CEF first with those patches, looks like you've already asked at https://www.magpcss.org/ceforum/viewtopic.php?f=6&t=19804
